### DISCUSSION TITLE
Why is this named as WSL rather than LSW?

### DISCUSSION THREAD

**Discussion by NeoForce0** [microsoft/WSL#7777] [Q&A] [answered]
I was looking at the given name for WSL(windows subsystem for Linux). But this system is reversed in my opinion and is more of a Linux subsystem for windows. Or maybe a Windows subsystem with integrated Linux (just to maintain WSL order)

**Comment by Biswa96** [comment, marked as answer]
Spell it like this, a subsystem in Windows to run Linux.
